HarpGuru/Controls/CovarianceSeries/index.ts

```diff
@@ -0,0 +1,9 @@
+export { getCovarianceSeries } from './getCovarianceSeries'
+export { CovariantMembers } from './types'
+export type {
+  CovariancePrimer,
+  HarpKeyCovariancePrimer, PozitionCovariancePrimer, RootPitchCovariancePrimer,
+  KeyByRootAtPozitionPrimer, KeyByPozitionAtRootPrimer,
+  PozitionByRootAtKeyPrimer, PozitionByKeyAtRootPrimer,
+  RootByPozitionAtKeyPrimer, RootByKeyAtPozitionPrimer,
```

Comment:
Do I use these subsubtypes outside of the module? If not I think I'll remove them from here.

Reply:
I've removed them. I feel like this might be a subject I'm being inconsistent on, but obviously there are conflicting pressures. On the one hand it's good to keep exports space clean, but on the other, it could be useful for drilling down in to problems to be able to be very specific about sub sub types. On this occasion I'm favouring the cleanliness since I can do that drilling down in the tests within the CovarianceSeries module itself.